oscarapi/views/basket.py

```diff
@@ -126,7 +127,11 @@ def add_voucher(request, format=None):
                                              context={'request': request})
     if v_ser.is_valid():
         basket = get_basket(request)
-        voucher = v_ser.object
```

Comment:
Did that become validated_data? But how do you get a model instance out of a modelserializet in rf3???

Is there really not a more direct way?